src/Components/Web.JS/src/Boot.WebAssembly.ts

```diff
@@ -39,15 +38,13 @@ async function boot(options?: any): Promise<void> {
 
   // Fetch the boot JSON file
   const bootConfig = await fetchBootConfigAsync();
-  const embeddedResourcesPromise = loadEmbeddedResourcesAsync(bootConfig);
 
   if (!bootConfig.linkerEnabled) {
     console.info('Blazor is running in dev mode without IL stripping. To make the bundle size significantly smaller, publish the application or see https://go.microsoft.com/fwlink/?linkid=870414');
   }
 
   // Determine the URLs of the assemblies we want to load, then begin fetching them all
-  const loadAssemblyUrls = [bootConfig.main]
-    .concat(bootConfig.assemblyReferences)
+  const loadAssemblyUrls = bootConfig.assemblyReferences
```

Comment:
why is it ok to remove `bootConfig.main`? does that show up in `assemblyReferences`? If that's the case should we rename `assemblyReferences` to `assemblies`?

Reply:
Yeah. It's the root of our search graph, so it always shows up. Changing it sounds perfectly fine.